Neos.Neos/Resources/Public/JavaScript/Interfaces/NeosI18n.ts

```diff
@@ -0,0 +1,4 @@
+export default interface NeosI18n {
+  translate: (id: string, fallback: string, packageKey: string, source: string, args: any[]) => string;
```

Comment:
Shouldn't it be the same as the I18nRegistry Interface in the Neos.Ui?

Reply:
The problem is that the signature is different in the I18nRegistry. So if we sync it, it will be breaking.
The I18nRegistry also does not use TS actually.

**I18nRegistry:**
```
translate(idOrig, fallbackOrig, params = {}, packageKeyOrig = 'Neos.Neos', sourceNameOrig = 'Main', quantity = 0)
```